pkg/api/orchestrators.go

```diff
@@ -181,12 +182,12 @@ func kubernetesInfo(csOrch *OrchestratorProfile) ([]*OrchestratorVersionProfile,
 func kubernetesUpgrades(csOrch *OrchestratorProfile) ([]*OrchestratorProfile, error) {
 	ret := []*OrchestratorProfile{}
 
-	currentVer, err := semver.NewVersion(csOrch.OrchestratorVersion)
+	currentVer, err := semver.Make(csOrch.OrchestratorVersion)
 	if err != nil {
 		return nil, err
 	}
-	nextNextMinorVersion := currentVer.IncMinor().IncMinor()
-	upgradeableVersions := common.GetVersionsBetween(common.GetAllSupportedKubernetesVersions(), csOrch.OrchestratorVersion, nextNextMinorVersion.String(), false, true)
+	nextNextMinorString := strconv.FormatUint(currentVer.Major, 10) + "." + strconv.FormatUint(currentVer.Minor+2, 10) + ".0"
```

Comment:
this is a workaround I had to apply because the `IncMinor()` function equivalent doesn't exist in this package. All it does is keep the same `Major`, increment `Minor` by 2 and set `patch` to 0. Validated that this works in unit tests.